iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -0,0 +1,140 @@
+
+        var iframes = $(iv._reparentDocument());
+
+        var taxonomyData = iv._getTaxonomyData();
+        if (taxonomyData === null) {
+            $('#ixv .loader .text').text("Error: Could not find viewer data");
+            $('#ixv .loader').removeClass("loading");
+            return;
+        }
+        var report = new iXBRLReport(JSON.parse(taxonomyData));
+        if (report.isDocumentSet()) {
+            var ds = report.documentSetFiles();
+            for (var i = 1; i < ds.length; i++) {
+                var iframe = $("<iframe />").attr("src", ds[i]).appendTo("#ixv #iframe-container");
+                iframes = iframes.add(iframe);
+            }
+            iv._checkDocumentSetBrowserSupport();
+        }
+
+        /* Poll for iframe load completing - there doesn't seem to be a reliable event that we can use */
+        var timer = setInterval(function () {
+            var complete = true;
+            iframes.each(function (n) {
+                var iframe = this;
+                var iframeDoc = iframe.contentDocument || iframe.contentWindow.document;
+                if ((iframeDoc.readyState != 'complete' && iframeDoc.readyState != 'interactive') || $(iframe).contents().find("body").children().length == 0) {
+                    complete = false;
+                }
+            });
+            if (complete) {
+                clearInterval(timer);
+                $('#ixv .loader .text').text("Building search index");
+
+                var viewer = new Viewer(iframes, report);
+
+                setTimeout(function () {
+                    inspector.setReport(report);
+                    inspector.setViewer(viewer);
+
+                    interact('#viewer-pane').resizable({
+                        edges: { left: false, right: ".resize", bottom: false, top: false},
+                        restrictEdges: {
+                            outer: 'parent',
+                            endOnly: true,
+                        },
+                        restrictSize: {
+                            min: { width: 100 }
+                        },
+                    })
+                    .on('resizestart', function (event) {
+                        $('#ixv').css("pointer-events", "none");
+                    })
+                    .on('resizemove', function (event) {
+                        var target = event.target;
+                        var w = 100 * event.rect.width / $(target).parent().width();
+                        target.style.width = w + '%';
+                        $('#inspector').css('width', (100 - w) + '%');
+                    })
+                    .on('resizeend', function (event) {
+                        $('#ixv').css("pointer-events", "auto");
+                    });
+                    $('#ixv .loader').remove();
+
+                    /* Focus on fact specified in URL fragment, if any */
+                    inspector.handleFactDeepLink();
+                },0);
+
+
```

Comment:
extra spaces here and below

Reply:
Fixed in 3a0f1bb88bcd871961c471695327c5ca3430bd9a